networkit/sampling.py

```diff
@@ -3,25 +3,43 @@
 __author__ = "Elisabetta Bergamini"
 
 def bfsSample(G, source=None, k = 50):
```

Comment:
It's strange to have this implemented in Python. In a separate PR we should implement this in C++ and move it to a dedicated module.